### ISSUE TITLE
Using example setup in readme generate errors

### ISSUE THREAD

**Issue by redirect11** [Badhi/nvim-treesitter-cpp-tools#25]
Hi, 
if I use your config example for treesitter like in the readme I get this error:
```
E5108: Error executing lua ...-cpp-tools/lua/nvim-treesitter/nt-cpp-tools/internal.lua:5: loop or previous error loading module 'nvim-treesitter.nt-cpp-tools.output_handlers'
stack traceback:
        [C]: in function 'require'
        ...-cpp-tools/lua/nvim-treesitter/nt-cpp-tools/internal.lua:5: in main chunk
        [C]: in function 'require'
        ...ker/start/nvim-treesitter-cpp-tools/lua/nt-cpp-tools.lua:65: in function 'init'
        [string ":lua"]:1: in main chunk
```

The solution is to not use the `custom_define_class_function_commands` field in the example. So if I remove this part:

```lua
custom_define_class_function_commands = { -- optional
            TSCppImplWrite = {
                output_handle = require'nvim-treesitter.nt-cpp-tools.output_handlers'.get_add_to_cpp()
            }
            --[[
            <your impl function custom command name> = {
                output_handle = function (str, context) 
                    -- string contains the class implementation
                    -- do whatever you want to do with it
                end
            }
            ]]
        }
```

the plugin loads correctly. What is wrong with that configuration?

**Comment by Badhi** [comment]
Hi, this configuration was added recently. To confirm, do you have the latest source files? especially the following file
```
nvim-treesitter-cpp-tools/lua/nvim-treesitter/nt-cpp-tools/output_handlers.lua
```

**Comment by redirect11** [comment]
I'm using [NvChad](https://github.com/NvChad/NvChad) . Could it be due to lazy-load stuff from it?

**Comment by Badhi** [comment]
Hi @redirect11 ,

I have made some changes in the configuration of the plugin to fix #26. Could you please retry it on your setup and see if the issue has been resolved?
